python/taichi/ui/camera.py

```diff
@@ -195,12 +197,14 @@ def get_projection_matrix(self, aspect):
         """
         return self.ptr.get_projection_matrix(aspect)
 
-    def track_user_inputs(self,
-                          window,
-                          movement_speed=1.0,
-                          yaw_speed=2,
-                          pitch_speed=2,
-                          hold_key=None):
+    def track_user_inputs(
+        self,
+        window,
+        movement_speed: float = 10.0,
```

Comment:
I suggest keeping the values to previously used so we don't break user code. See comments below for speed adjustments

Reply:
I have tested & committed the change. Please let me know if I made anything wrong.